ts/masters-datagrid/datagrid.src.ts

```diff
@@ -183,6 +189,8 @@ ValidatorComposition.compose(G.Table);
 CellRenderersComposition.compose(G.Column);
 
 G.CellRendererRegistry = G.CellRendererRegistry || CellRendererRegistry;
+G.CellContentPro = CellContentPro;
+G.CellRenderer = CellRenderer;
```

Comment:
If we have to expose things here, there is a high probability that we also need to do it in `datagrid.src.d.ts` in the `gulptasks` folder (the mock dts for the bundle).